draft-ietf-quic-qpack.md

```diff
@@ -316,40 +311,39 @@ the input header block.
 
 Upon receipt of a header block, the decoder examines the Required Insert Count.
 When the Required Insert Count is less than or equal to the decoder's Insert
-Count, the header block can be processed immediately.  Otherwise, the stream is
-blocked.
+Count, the header block can be processed immediately.  Otherwise, the stream on
+which the header block was received becomes blocked.
 
 While blocked, header block data SHOULD remain in the blocked stream's flow
 control window.  A stream becomes unblocked when the Insert Count becomes
 greater than or equal to the Required Insert Count for all header blocks the
 decoder has started reading from the stream.
 <!-- doesn't the stream become unblocked when the encoder receives the acks? -->
 
-If the decoder encounters a header block with a Required Insert Count value
-larger than defined in {{blocked-streams}}, it MAY treat this as a connection
-error of type HTTP_QPACK_DECOMPRESSION_FAILED.  If the decoder encounters a
-header block with a Required Insert Count value smaller than defined in
-{{blocked-streams}}, it MUST treat this as a connection error of type
-HTTP_QPACK_DECOMPRESSION_FAILED as prescribed in {{invalid-references}}.
+When processing header blocks, the decoder expects the Required Insert Count
+value to exactly match the value defined in {{blocked-streams}. If it encounters
+a smaller value, it MUST treat this as a connection error of type
+HTTP_QPACK_DECOMPRESSION_FAILED (see {{invalid-references}}). If it encounters a
+larger value, it MAY treat this as a connection error of type
+HTTP_QPACK_DECOMPRESSION_FAILED.
```

Comment:
I worry slightly that people might get confused about which "value" is smaller or larger than which other "value", because that term is used to describe both.